packages/dds/matrix/src/test/memory/matrix.spec.ts

```diff
@@ -34,95 +34,119 @@ describe("Matrix memory usage", () => {
 		// See the comment at the top of the test suite for more details.
 	});
 
-	benchmarkMemory(
-		new (class implements IMemoryTestObject {
-			title = "Create empty Matrix";
-			minSampleCount = 500;
-
-			private localMatrix: SharedMatrix = createLocalMatrix("testLocalMatrix");
-
-			async run() {
-				this.localMatrix = createLocalMatrix("testLocalMatrix");
-			}
-		})(),
-	);
-
-	const numbersOfEntriesForTests = [100, 1000, 10_000];
-
-	numbersOfEntriesForTests.forEach((x) => {
+	describe("Detached Matrix", () => {
 		benchmarkMemory(
 			new (class implements IMemoryTestObject {
-				title = `Insert and remove ${x} columns`;
+				title = "Create empty Matrix";
+				minSampleCount = 500;
+
 				private localMatrix: SharedMatrix = createLocalMatrix("testLocalMatrix");
 
 				async run() {
-					for (let i = 0; i < x; i++) {
-						this.localMatrix.insertCols(0, 100);
-						this.localMatrix.removeCols(0, 100);
-					}
-				}
-
-				beforeIteration() {
 					this.localMatrix = createLocalMatrix("testLocalMatrix");
 				}
 			})(),
 		);
 
-		benchmarkMemory(
-			new (class implements IMemoryTestObject {
-				title = `Insert and remove ${x} rows`;
-				private localMatrix: SharedMatrix = createLocalMatrix("testLocalMatrix");
+		const numbersOfEntriesForTests = [100, 1000, 5000];
```

Comment:
What's the motivation for doing this in 3 batches?  Is there something to observe in the small batches that we wouldn't learn in the large batches?

Reply:
Nothing specific I can point to, tbh; when I first wrote these (or maybe others that inspired these) my thinking was "let's look at DDS behavior under light usage, medium usage, and heavy usage; we might notice different things in each case (e.g. a memory leak that isn't obvious in the light usage case but becomes noticeable if we push the DDS further)" in an admittedly hand-wavy way.

I'll say we did end up seeing different behaviors for different batches _once_... but in that case it seems to have been an artifact of the measuring approach (which hasn't been addressed).